cdc/processor/pipeline/main_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	"testing"
+
+	"github.com/pingcap/tiflow/pkg/leakutil"
+	"go.uber.org/goleak"
+)
+
+func TestMain(m *testing.M) {
+	leakutil.SetUpLeakTest(
+		m,
+		goleak.IgnoreTopFunction("github.com/pingcap/tiflow/cdc/sorter/unified.newBackEndPool.func1"),
```

Comment:
It is better to add the `IgnoreTopFunction` here:https://github.com/pingcap/tiflow/blob/master/pkg/leakutil/leak_helper.go#L26 
and call the `SetUpLeakTest` .

Reply:
not changed, just copy from sorter_test